tests/endpoints/test_canal_transit_real.py

```diff
@@ -6,8 +6,3 @@
 class TestCanalTransitReal(TestCaseUsingRealAPI):
     def test_default_search(self):
         CanalTransit().search()
-
-    def test_search_for_lng_vessels(self):
-        df = CanalTransit().search(filter_vessel_classes=["lng"]).to_df()
-
-        assert list(df.columns) == DEFAULT_COLUMNS
```

Comment:
Removing this flaky test after discussing with jb
https://vortexa.slack.com/archives/CD5MA408P/p1712756495081709